backend/lib/plugins/socket.js

```diff
@@ -437,6 +437,11 @@ function onSocketConnect(socket) {
     );
   });
 
+  socket.on("CLEAR_NOTIFICATION", async (data) => {
+    const { notificationId } = data;
+    await Notification.findByIdAndUpdate(notificationId, { isCleared: true })
+  });
+
```

Comment:
For security reasons, I would also add `userId` to the query to make sure the user owns that notification.
But I think since no one has your notifications ids we can sacrifice this for performance.
What do you think @mannykary